encoding/base58.ts

```diff
@@ -0,0 +1,383 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+const mapBase58 = [
```

Comment:
I would consider using a deno-no-lint directive to stop this array from being split up like this, and write it in a 16x16 grid. It would look cute.

Reply:
Could also be done, I'll open the PR with the refactor and include this change